src/store/cache/user/graphHooks.ts

```diff
@@ -0,0 +1,457 @@
+import { useQuery, gql } from '@apollo/client'
+import { useDispatch } from 'react-redux'
+import { ThunkAction } from 'redux-thunk'
+import { Action } from 'redux'
+import BN from 'bn.js'
+import { getUserProfile as get3BoxProfile } from 'services/3box'
+import { Status, User, Vote, ServiceType, Operator } from 'types'
+import Audius from 'services/Audius'
+import { AppState } from 'store/types'
+import { setLoading, setUsers } from './slice'
+import { useEffect, useState } from 'react'
+
+const formatUser = async (
+  aud: Audius,
+  user: FullUser
+): Promise<User | Operator> => {
+  const userWallet = aud.toChecksumAddress(user.id)
+  const profile = await get3BoxProfile(userWallet)
+
+  // TODO: Make this part faster
```

Comment:
why/how? (add to comment so we dont forget)

Reply:
Just by async loading in the profiles, but idk how it would work from a UX point of view. Also by actually implementing pagination 🙃